applications/operators/olm-status.adoc

```diff
@@ -0,0 +1,15 @@
+[id="olm-status"]
+= Viewing Operator status
+include::modules/common-attributes.adoc[]
+:context: olm-status
+
+toc::[]
+
+Understanding the state of the system in Operator Lifecycle Manager (OLM) is
+important for making decisions around and debugging problems with installed
```

Comment:
I wonder if this would read better as:
_making decisions about and debugging problems with..._

Reply:
Yes, better! This was tripping me up too.